Doc/whatsnew/3.9.rst

```diff
@@ -426,6 +426,11 @@ Removed
   is opened.
   (Contributed by Victor Stinner in :issue:`39357`.)
 
+* The *encoding* parameter of :func:`json.loads` has been removed.  Since
+  Python 3.1, it was deprecated and ignored.  And using it was emitting
+  :exc:`DeprecationWarning` since Python 3.8.
```

Comment:
```
Python 3.1, it was deprecated and ignored. As of Python 3.8, using it
emitted a :exc:`DeprecationWarning`.
```

Reply:
I'm not good at English so I don't understand the difference between current sentence and your suggestion.
I just copied upper entry and edit.  If I should change "using it was emitting..." to "using it emitted", shoud I change the upper entry too?

Reply:
Yeah, "emitted" is better since we're talking about a past version. Feel free to change `was emitting -> emitted` on the entry above as well.